tpu/utils/README.md

```diff
@@ -0,0 +1,24 @@
+# Utilities
+
+This directory contains helpful utilities to use [Cloud TPU](https://cloud.google.com/tpu/).
+
+
+## TPU Utilities
+
+[`tpu_utils.py`](tpu_utils.py) contains Python wrapper functions for the [Cloud TPU REST API](https://cloud.google.com/tpu/docs/reference/rest/) to manage the TPU resources.  This allows programatically creating, deleting, and monitoring TPU nodes.
+
+
+## TPU Survival Training
+
+[`tpu_survival_job.py`](tpu_survival_job.py) is a simple script for resuming training when a [preemptible TPU](https://cloud.google.com/tpu/docs/preemptible) is preempted during a training session.
+
+To use it:
+
+1. Copy (or Git clone) the files in this folder to a VM instance on Compute Engine that has access to TPUs.
+
+1. Modify the `submit_preemptible.sh` script for your training job.  
```

Comment:
What does submitting a preemptible VM do?
Doe that mean the VM is pre-emptible or we are now making it pre-emptible?

Reply:
In this case the preemption is on the TPU node.  The node being preemptible means it will be automatically shut down by the platform (and not by the user) if the resources are needed for non-preemptible workloads (by other users).

Preemptible TPUs are at a lower pricing point.

Reply:
Yes, but it is to be used with `tpu_survival.py`, which creates subprocesses calling the shell script.